src/api/portfolioFunctions.js

```diff
@@ -0,0 +1,103 @@
+import { SafePromiseAll } from './helpers'
+import { goatLowestPrice } from './dataSources'
+import createRequestObject from './createRequest'
+
+export async function getPortfolioData (userID, currency, location) {
+  try {
+    const request = createRequestObject('portfolio_get', {
+      user_id: userID,
+      currency
+    })
+    const response = await fetch(request.url, request.headers)
+
+    let portfolio = await response.json()
+    portfolio.forEach(item => { item.data = JSON.parse(item.data) })
+
+    // Get current prices for each item
+    const reqs = portfolio.map(item => itemWithCurrentPrice(
```

Comment:
shitty variable name, this is a list of api requests right? Could be better as `apiRequests` or sumn

Reply:
I've lost sleep trying to find a name for this variable lol. I found it like that so I just decided to not change it.
`apiRequests` doesn't change it much but I can try something.

Reply:
made it `itemsWithCurrentPricePromises`